src/components/SingleArticleCard.jsx

```diff
@@ -37,8 +52,29 @@ function SingleArticleCard() {
       <p>Author: {article.author}</p>
       <p>Topic: {article.topic}</p>
       <p>Content: {article.body}</p>
+      <p>
+        Posted At: {dayjs(article.created_at).format("HH:mm:ss - DD-MM-YYYY")}
+      </p>
       <p>Votes: {article.votes}</p>
-      <p>Posted At: {article.created_at}</p>
+      <button
+        className="increase-vote-button"
+        onClick={(e) => {
+          handleChange(1);
+          e.target.disabled = true;
+        }}
+      >
+        Vote Up
+      </button>
+      <button
+        className="decrease-vote-button"
+        onClick={(e) => {
+          handleChange(-1);
```

Comment:
Great refactor! You don't need a handleIncrement and handleDecrement function when you can just pass either a positive or negative number
One small note is I think the name of this could be a bit clearer - perhaps `handleVoteChange`?